packages/plugin/src/Bundles/Form/HiddenInputs/CsrfInput.php

```diff
@@ -17,21 +18,13 @@ public function __construct()
             return;
         }
 
-        if (!\Craft::$app->request->isConsoleRequest) {
-            // Prevent response from being cached with token
-            \Craft::$app->getResponse()->setNoCacheHeaders();
-        }
-
         Event::on(Form::class, Form::EVENT_RENDER_AFTER_OPEN_TAG, [$this, 'attachInput']);
         Event::on(Form::class, Form::EVENT_OUTPUT_AS_JSON, [$this, 'attachToJson']);
     }
 
     public function attachInput(RenderTagEvent $event)
     {
-        $name = \Craft::$app->getConfig()->getGeneral()->csrfTokenName;
-        $token = \Craft::$app->request->csrfToken;
-
-        $event->addChunk('<input type="hidden" name="'.$name.'" value="'.$token.'" />');
+        $event->addChunk(Html::csrfInput());
```

Comment:
Allows it to work with `asyncCsrfInputs` https://craftcms.com/docs/5.x/reference/config/general.html#asynccsrfinputs